docs/src/content/docs/es/guides/components.mdx

```diff
@@ -101,3 +101,28 @@ Utiliza una cuadrícula de tarjetas en tu página de inicio para mostrar las car
 ```
 
 :::
+
+### Iconos
+
+import { Icon } from '@astrojs/starlight/components';
+import IconsList from '../../../components/icons-list.astro';
```

Comment:
Thanks @Waxer59!

I think we need another level of `../` here to import correctly:

```suggestion
import IconsList from '../../../../components/icons-list.astro';
```

Reply:
Nice catch Chris 🚀